Tools/CISupport/ews-build/steps.py

```diff
@@ -1720,23 +1720,29 @@ class CloseBug(buildstep.BuildStep, BugzillaMixin):
     flunkOnFailure = False
     haltOnFailure = False
 
-    def start(self):
-        self.bug_id = self.getProperty('bug_id', '')
-        if not self.bug_id:
-            self._addToLog('stdio', 'bug_id build property not found.\n')
-            self.descriptionDone = 'No bug id found'
-            self.finished(FAILURE)
-            return None
+    def __init__(self, **kwargs):
+        self.bug_id = ''
+        super(CloseBug, self).__init__(**kwargs)
 
+    def start(self):
+        self.bug_id = self.getProperty('bug_id', self.bug_id)
         rc = self.close_bug(self.bug_id)
         self.finished(rc)
         return None
 
     def getResultSummary(self):
+        if self.results == SKIPPED:
```

Comment:
Actually it's better not to have this custom result summary for SKIPPED case. First of all it wouldn't be displayed in the buildbot UI, since we set hideStepIf as well. Secondly, this custom string will show up in status-bubble pop-over message, while the default one (ending in `(skipped)`)  is hidden (See: https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-app/ews/views/statusbubble.py#L70)